lib/DataRepeater.js

```diff
@@ -471,6 +473,17 @@ var DataRepeater = module.exports = kind(
 				p.removeListener(e, this._handlers[e]);
 			}
 		}
+		if(c && this.selectionProperty) {
+			var selectionProperty = this.selectionProperty,
```

Comment:
While we're at it, let's move these `var`s up to the top of the method. Might consider defining these methods at the top of the module (like `at()`).